lib/ui/myplot/my_plot_child_item.dart

```diff
@@ -37,7 +36,7 @@ class MyPlotListItem {
                   child: Column(
                     crossAxisAlignment: CrossAxisAlignment.start,
                     children: <Widget>[
-                      Text(cropsData.name ?? Strings.myPlotItemDefaultTitle,
+                      Text("TESTING" ?? Strings.myPlotItemDefaultTitle,
```

Comment:
I would use the view model data, not a string literal here. (for testing we can inject a mock view model) the concept of defaults will be implemented in the view model so this will just delegate to that and will not need the default string here.